genai-perf/genai_perf/config/input/config_endpoint.py

```diff
@@ -82,7 +82,7 @@ def parse(self, endpoint: Dict[str, Any]) -> None:
             if key == "model_selection_strategy":
                 self.model_selection_strategy = ModelSelectionStrategy(value.upper())
             elif key == "backend":
-                self.backend = OutputFormat(value.upper())
+                self.backend = OutputFormat(value.upper()) if value else None
```

Comment:
When would it be None? Would that be an error if a backend isn't provided with the field? It seems like this should be something that's universal (i.e. all/most args do a check like this or none of them do).

Reply:
My understanding is that with the latest changes the backend can now be left blank in the config and this was causing a problem for me when I cleared the default value (trtllm) from the template.